index.html

```diff
@@ -203,6 +203,51 @@ <h3>Skills</h3>
           </div>
         </section>
       </section>
+      <section id="form-section">
+        <h2 class="form-heading">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+        <form action="https://formspree.io/f/mdovqere" method="post">
+          <label for="name">
+            <input id="name" name="name" type="text" maxlength="30" placeholder="Full name"required>
+          </label>
+          <label for="email">
+            <input id="email" name="email" type="email" placeholder="Email address" required>
+          </label>
+          <label for="message">
+            <textarea name="message" maxlength="500" required>Write me something...</textarea>
+          </label>
+          <button type="submit">Get in touch</button>
```

Comment:
- [ ] Well done for adding the `hover` interaction 🥇Please make sure to add the button interaction for an `active` state (when it is pressed) as mentioned in the project requirements 

  > _Implement the button interactions (enable, hover, etc.)._

Your design | Given design |
---| ---|
![image](https://github.com/tajemouti/portfolio/assets/106119105/ccec95bd-2cbe-4a79-97bf-35c13e9e05fe) | ![image](https://github.com/tajemouti/portfolio/assets/106119105/a0d09cfd-e292-4d5a-a3c5-0c86cd845a32)